migrations/m201013_172104_add_calendar_entry_location.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m201013_172104_add_calendar_entry_location
+ */
+class m201013_172104_add_calendar_entry_location extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn('calendar_entry', 'location', 'varchar(64) DEFAULT NULL');
```

Comment:
Just out of curiosity the limit of 64 characters is this some kind of ics standard or did you randomly choose this limit?

Reply:
I wanted to ask but forgot, yes I am not sure 64 will be enough for location, what do you think 255 or more?
This field looks unlimited on macos Calendar app because there I can type very long value.